src/problems/cvrp/local_search/local_search.cpp

```diff
@@ -494,6 +495,20 @@ void cvrp_local_search::try_job_additions(const std::vector<index_t>& routes,
         }
       }
 
+      auto smallest = std::numeric_limits<gain_t>::max();
```

Comment:
Well done for using smallest and second smallest, that's a clever way to get rid of the inner loop!